libs/fn/src/lib/cdk/selectable-list/SelectComponentRootToken.ts

```diff
@@ -0,0 +1,9 @@
+import { Observable } from 'rxjs';
```

Comment:
dash-case-naming instead of UpperCamelCase, please.

Reply:
In case of the interfaces too?

Reply:
I was thinking about having also only [fnSelectable] in case of grouping you know. Every item could have been as well as root, as well as item. So that they would resolve children and add selection logic to them. On self selection it would select probably itself and all of it's children or something like that. Like a Tree